pkg/operator/api/api.go

```diff
@@ -74,11 +75,11 @@ func (a *apiServer) GetConfiguration(ctx context.Context, in *operatorv1pb.GetCo
 	key := types.NamespacedName{Namespace: in.Namespace, Name: in.Name}
 	var config configurationapi.Configuration
 	if err := a.Client.Get(ctx, key, &config); err != nil {
-		return nil, fmt.Errorf("error getting configuration: %s", err)
+		return nil, pkgerror.Wrap(err, "error getting configuration")
```

Comment:
Example of how this works can be found at.  https://play.golang.org/p/781Sl7j8I-X , please have a look it will help us get on same page.